common/src/path.rs

```diff
@@ -442,6 +498,9 @@ mod tests {
         assert_eq!(shorten_path("/foobarbaz/blahblah", 16), ".../blahblah");
         assert_eq!(shorten_path("/foobarbazblahblah", 12), "...lahblah");
         assert_eq!(shorten_path("", 0), "");
+
+        assert_eq!(shorten_path("아이쿱 조합원 앱카드", 9), "아...");
+        assert_eq!(shorten_path("아이쿱 조합원 앱카드", 20), "...ᆸ카드");
```

Comment:
is this a right-to-left language? I’m confused why the result is flipped.

Reply:
This is just the way the function works, it's what I meant with "odd" behavior. For short `length <= 10`, it truncates in the back, and for `length > 10` it truncates in the front.  I'm planning to change it in the next step, but for now this PR should only fix the panics.

Reply:
ah, I should have paid more attention then ;-) nvm